dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -643,16 +615,10 @@ def expect_controller_logs_milestone_regexp(regexp)
       end
     end
 
-    # record activity in session
-    assert_equal 100, client_state.level_progress(@script_level.level_id)
-
-    # record the total lines of code in session
-    assert_equal 20, client_state.lines
-
     assert_response :success
 
     expected_response = build_expected_response(
-        total_lines: 20,
+        total_lines: 0,
```

Comment:
I don't think total_lines is used on the server end for anonymous users. There's a call to track_progress_for_user (signed in user) and track_progress_in_session (signed out user), the latter is used and does not do anything with total_lines